user_menager.py

```diff
@@ -0,0 +1,56 @@
+from user import User
+from typing import List
+from db_menager import DBManager
+
+
+class UserManager(DBManager):
+    def __init__(self) -> None:
+        self.connection = self.get_connection()
+        self.users = self._read_users()
+
+    def _read_users(self) -> List[User]:
+        users = []
+        mycursor = self.connection.cursor()
+        mycursor.execute("SELECT * FROM users")
+        data_users = mycursor.fetchall()
+
+        for data in data_users:
+            user = User(data[1], data[2], data[3], data[0])
+            users.append(user)
+
+        return users
+
+    def get_user(self, login):
```

Comment:
add hints as everywhere
make your code as clean as you could, imagine you work at Google, take this project as much seriously as you can